package.json

```diff
@@ -165,6 +165,7 @@
     "minify-stream": "2.0.1",
     "minimist": "^1.2.5",
     "mkdirp": "0.5.1",
+    "moment-timezone": "^0.5.31",
```

Comment:
we were previously relying on moment that was implicitly installed by expo for "@uppy-example/react-native-expo" I think (and it only worked because moment is a global object)

Reply:
sounds good, this is updated